hivemq-edge/src/frontend/src/modules/DomainOntology/components/charts/HierarchicalEdgeBundling.tsx

```diff
@@ -0,0 +1,119 @@
+import { useMemo } from 'react'
+import { hierarchy, cluster } from 'd3-hierarchy'
+import { lineRadial, curveBundle } from 'd3-shape'
+// import { interpolateRdBu } from 'd3-scale-chromatic'
+import { Tree, TreeLeaf } from '@/modules/DomainOntology/types.ts'
+
+type HierarchicalEdgeBundlingProps = {
+  width: number
+  height: number
+  data: Tree
+}
+
+const BUNDLE_COEFF = 0.95
+const MARGIN = 200
+
+export const HierarchicalEdgeBundling = ({ width, height, data }: HierarchicalEdgeBundlingProps) => {
+  const hierarchyxxx = useMemo(() => {
```

Comment:
suggestion: remove the `xxx` from the variable name

```suggestion
  const hierarchy = useMemo(() => {
```